README.md

```diff
@@ -1,5 +1,9 @@
 # Vault on GCE Terraform Module
 
+**IMPORTANT:** this is a legacy module that will be deprecated once their dependencies are gone
```

Comment:
```suggestion
**IMPORTANT:** this is a legacy module that will be deprecated once it is no longer depended on.
```

Reply:
Nothing better that a review of a native speaker! :-)

Reply:
I am constantly impressed by Neidson's, and Andres', and your english. Happy to help.